src/sa_web/static/js/views/place-form-view.js

```diff
@@ -17,15 +18,24 @@ var Shareabouts = Shareabouts || {};
       // Bind model events
       this.model.on('error', this.onError, this);
     },
-    render: function(){
+    render: function(category, category_selected){      
       // Augment the model data with place types for the drop down
+
+      if (category != undefined) {
+        S.TemplateHelpers.insertInputTypeFlags(this.options.placeConfig.categories[category].fields);
+      }
+      
       var data = _.extend({
         place_config: this.options.placeConfig,
+        // when category is defined, the selected_category property will contain config file details for fields in the given category
+        selected_category: this.options.placeConfig.categories[category],
         user_token: this.options.userToken,
-        current_user: S.currentUser
+        current_user: S.currentUser,
+        category_selected: category_selected || false
```

Comment:
Can we change the `category_selected` key and parameter to `is_category_selected`? I think it would be a little more clear that it's a boolean, and avoid confusion with `selected_category`.